katalog/fluentd/README.md

```diff
@@ -1,9 +1,11 @@
-# Fluentd
+# Fluentd & Fluent-bit
 
 Fluentd is an open source data collector for unified logging that can store
 collected data in various destinations. Fluentd deployment in Fury Kubernetes
 Logging is configured to write records into Elasticsearch.
 
+Fluent-bit is an open source log collector and forwarder. It sends all collected logs to fluentd. TODO
```

Comment:
what's to be done?

Reply:
I didn't finish writing the README, it's a placeholder 😊